qa/automation/CareerPortal.Tests.Scenarios/1. Search by Filters/1.1. Search by Filters.feature

```diff
@@ -0,0 +1,134 @@
+    Given User opened https://career.quantori.com/ru/positions/cloud-python-engineer website
+    When At the bottom of the page user push "Back to open job positions" button
+    Then User should be redirected to the new page https://career.quantori.com/ru/positions
+    And  User should see default page "Quantori open job positions"
+
+Scenario: 1.1_03 Checking the search by 1 competence filter
+    When User chooses "Databases" competence with list box filter Competence
+    And User pushes "search" button
+    Then User should be redirected to the new page https://career.quantori.com/ru/positions
+    And User should see only 2 vacancy "Data Engineer (SQL)" and "Snowflake Data Warehouse Certified Engineer"
+
+Scenario Outline: 1.1_04 Checking the search by 1 technology filter
+    When User chooses "SQL" technology with list box filter Technology
+    And User pushes "search" button
+    Then User should be redirected to the new page https://career.quantori.com/ru/positions
+    And User should see only 8 <Position>
+    
+    Examples: 
+                |Position                                              |
+                |Junior JavaScript Engineer                            |
+                |Senior/Lead Manual QA Engineer (Life Science projects)|
+                |Senior/Lead Manual QA Engineer (internal projects)    |
+                |Data Engineer (SQL)                                   |
+                |Senior DWH / ETL Data Engineer                        |
+                |Data Engineer (Spark)                                 |
+                |Junior JavaScript Engineer                            |
+                |Senior QA Automation (Python) Engineer / SDET         |
+                |Senior Data Engineer                                  |
+
+
+Scenario: 1.1_05 Checking the search by 1 competence filter and 1 technology filter
+    When User chooses "Databases" competence with list box filter
+    And User chooses "SQL" technology with list box filter
+    Then User should be redirected to the new page https://career.quantori.com/ru/positions
+    And User should see only 1 vacancy "Data Engineer (SQL)"
+
+Scenario: 1.1_06 Checking the link with the selected components and technologies 
+    When User opened competence list box filter 
+    And User Chooses Databases competence 
+    And User opened technologies list box filter 
+    And User Chooses SQL technology
+    And User Copies link with filters https://career.quantori.com/ru?competences=Databases&technologies=SQL
+    And User Opens a new page in his browser 
+    And pastes the copied link
+    Then User should be redirected to the new page https://career.quantori.com/ru?competences=Databases&technologies=SQL
+    And User should see only 1 vacancy "Data Engineer (SQL)"
+
+Scenario: 1.1_07 Checking the form "Open job positions" without filters
+    Given https://career.quantori.com/ru/positions website is opened 
```

Comment:
Better to be on the main page and then click on that button and check opened page